.circleci/config.yml

```diff
@@ -159,8 +207,39 @@ jobs:
           path: /go/out/logs
 
   # Verify a-la-carte with no certificates ( smallest possible install, and cases without certs/mtls)
-  noauth:
-    <<: *integrationRemote
+  noauth_one_namespace:
+    <<: *integrationRemoteOneNamespace
+
+    # Can run multiple images, first is primary, all use localhost
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          command: |
+            mkdir -p $GOPATH/tmp
+            mkdir -p $GOPATH/src/istio.io/
+            git clone https://github.com/istio/istio.git $GOPATH/src/istio.io/istio
+            git clone https://github.com/istio/tools.git $GOPATH/src/istio.io/tools
+      - run:
+          command: make info dep prepare
+      - run:
+          command: make sync
+      - run:
+          command: make docker-run-test TEST_TARGET="run-test-noauth-micro"
+      - run:
+          command: make docker-run-test TEST_TARGET="run-test-noauth-full"
+      - run:
+          when: always
+          command: make kind-logs
+      - store_test_results:
+          path: /go/out/tests
+      - store_artifacts:
+          path: /go/out/logs
+
+  # Verify a-la-carte with no certificates ( smallest possible install, and cases without certs/mtls)
+  noauth_isolated_namespaces:
```

Comment:
I don't think we need this - the purpose of noauth is to verify the config used by knative and few others, as well as the case where user deploys istio gateway in their own namespace (next to the app). 

It also serves to test the case where dedicated pilots are deployed for gateway only, to minimize risks and improve 
segmentation.

If there is user demand for pilot in 1 ns, and ingress in a different ns - we can add it, but I don't want to add 
profiles without a clear demand, there are more useful configs we can test.

Reply:
Thanks, got it!